civihr_employee_portal/js/reports.js

```diff
@@ -919,6 +919,67 @@
                         }
 
                         this.$DOM.sections.dataWrapper.fadeIn();
+                        var $DOM = this.$DOM;
+
+                        $.ajax({
+                            'type': 'GET',
+                            'url': this.getJsonUrl() + ( typeof path !== 'undefined' ? path : '' ),
+                            'success': function(data) {
+                                var mappedResults = {},
+                                    key,
+                                    countRow = $('<tr />'),
```

Comment:
For any jQuery DOM object, please use a variable name that starts with a `$`, so they're easier to recognize
```js
    $countRow = $('<tr />'),
    $headerRow = $('<tr />'),
    $percentageRow = $('<tr />'),
```